content.css

```diff
@@ -69,6 +69,11 @@
 	display: none !important;
 }
 
+/* Hide "Topics to follow" box */
+.bt--notopics [aria-label="Timeline: "] {
```

Comment:
"Topics to follow" changes DOM structure over time. Perhaps when you implemented this the selector matched, likely due to an incomplete implementation on Twitter.com, but it no longer matches.

The ARIA label for "Topics to follow" is `Timeline: Carousel`. But that's too generic. It identifies other content displayed as a carousel, not just topics to follow. Implementing it as such has unwanted side-effects.

In general, try to avoid language-specific matching as much as possible. This prevents features from working in other languages. See issue #3

Reply:
Makes sense. Thank you for reviewing it.